scripts/config

```diff
@@ -210,6 +210,43 @@ ynh_app_config_apply() {
 
     _ynh_app_config_apply
     
+    # Activate captive portal or not
+    captive_portal=$(ynh_app_setting_get --app=$app --key=captive_portal)
+    if [[ "$captive_portal" =~ 1 ]]
+    then
+        echo "location / {" > /etc/nginx/conf.d/default.d/redirect_to_admin.conf
+        if [[ "$captive_portal__1" == "1" ]]
+        then
+            echo "    if (\$remote_addr ~ "^$ip4_nat_prefix__1.\d+$") {" >> /etc/nginx/conf.d/default.d/redirect_to_admin.conf
+
+            echo "      return 302 $captive_portal_url__1;" >> /etc/nginx/conf.d/default.d/redirect_to_admin.conf
+
+            echo "    }" > /etc/nginx/conf.d/default.d/redirect_to_admin.conf
+        fi
```

Comment:
Eeeeeh manually editing the nginx conf is bad™ ... it should really be handled through a regen conf hook, otherwise it's gonna be flagged as manually modified, which  the diagnosis will complain about, and then people will naively run the suggested `regen-conf --force` command, and there is apparently no way to revert the change when the captive portal stuff is disabled (and also the CI probably won't allow it anyway)